### PR TITLE
[DX-937] Update quick-start for Tyk Cloud to mention manual access approval re…

### PR DESCRIPTION
**Pull request by dcs3spp**
[DX-937](https://tyktech.atlassian.net/browse/DX-937)
[preview](https://deploy-preview-3855--tyk-docs.netlify.app/docs/nightly/deployment-and-operations/tyk-cloud-platform/quick-start)

[DX-937]: https://tyktech.atlassian.net/browse/DX-937?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by SonjaChevre** [comment]
hey @dcs3spp thanks for jumping on this quickly!

I'm not sure this is the right place to put this information - as this problem is about the classic portal. And we have two different portals at the moment (enterprise portal and classic portal). 

We probably need at last a page about deploying and managing enterprise portal in Tyk Cloud, maybe another one about classic portal. 

I have entered an item on the board for our next strategic product sync on Friday to discuss in the group, and will update you here after this meeting.

Sonja

File: tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md [hunk 1]
```diff
@@ -56,6 +56,12 @@ With your Tyk Cloud account set up, it's time to create your first API:
 
 Now that your API is created, you can explore and manage it through the Tyk Dashboard.
 
+{{< warning success >}}
+**Warning**
+
+To setup an API with Manual Access Approval enabled the configuration for sending emails is a manual process. To enable this feature please contact your account manager.
```
Comment (letzya) [inline tyk-docs/content/deployment-and-operations/tyk-cloud-platform/quick-start.md:62]: @dcs3spp 
1. do we need this pr or shall we close it
2. the sentence is too wordy and long,  couldn't understand it

**Comment by dcs3spp** [comment]
Closing until future sprint
